catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/query/data/model/QueryBasic.java

```diff
@@ -76,7 +76,7 @@
   @SerializedName("enterprise")
   private Boolean enterprise;
 
-  @SerializedName("sources")
+  @SerializedName(value="sources", alternate="src")
```

Comment:
This is an okay temporary fix but it should just be one values. Can we just change the QueryBasic model to use `src` instead so it's consistent with the CQL endpoint?

Reply:
Can you confirm that no one else is using this endpoint in the other downstreams? I figured they're not, but I can't confirm it.

Reply:
@bellcc

Reply:
For posterity, @bellcc said it looks like it isn't being used in a way that will hurt anyone if it's changed to "src". Making the change.